spotipy/oauth2.py

```diff
@@ -316,17 +316,18 @@ def get_authorize_url(self, state=None):
 
         return "%s?%s" % (self.OAUTH_AUTHORIZE_URL, urlparams)
 
-    def parse_response_code(self, url):
```

Comment:
public method that we must keep because others might be using it https://github.com/search?l=Python&q=%22parse_response_code%22+spotipy&type=Code